README.md

```diff
@@ -240,125 +233,37 @@ navigator.login.setStatus("logged-in", {
-  identity: {
-    providers: [{type: "indie-auth"}]
-  }
+  }]
 });
 ```
 
 ## Open Questions
 
 * Should we allow defining the profile information and API config in the `Login-Status` header?
 * Does expiration also expire the IdP-supplied `apiConfig` or just the account information?
+* Should it be possible to specify expiration times on a per-account basis?
+* Should it be possible to add or remove individual accounts from the list instead of replacing the entire account list?
 * Does `Login-Status: logged-out` clear the IdP-supplied `apiConfig?`
 * Should the API config be supplied using a call to the IdP registration API instead?
-* Given the lower barrier to entry and risk of abuse, should we support passive mode for Lightweight FedCM at all?
-* Under what conditions is presenting IdP picker mandatory vs UA-implementation-defined?
 * For `approved_clients`, should we support matching on the RP origin, not just the clientId?
 
-### Allowing the relying party to control credentials that appear in the credential chooser
-
-Since any site can claim to be an identity provider with any `"effectiveType"`, we may want to allow websites further control over the elements in the UI.
-However this carries a risk of information leak to the relying party of all of the origins of a given type.
-Currently the relying party may mitigate this by validating the origin of the returned credential, or by attempting to use the credential, and by repeating the authentication process if it is unacceptable.
-Here is an example of such behavior in some abstracted Javascript:
-
-```js
-while (true) {
-  let cred = navigator.credentials.get(options);
-  if (allowedOrigin(cred.origin) && credentialWorks(cred)) {
-    break;
-  }
-}
-useCredential(cred);
-```
-
-
 ## Detailed design discussion
 
 ### A light touch from the browser
 
-One core principal of this design is to get out of the identity provider's way as quickly and as much as possible. The purpose of UI when using this API should be to gather user consent to the linking of information between sites and then doing no more. Account selection, account data storage, policy presentation, and capability selection are all things we do not want to do as a browser as they are difficult and there is already an industry dedicated to solving these challenges. As such, each credential represents a connection to an identity provider, not an identity.
+One core principal of this design is to get out of the identity provider's way as quickly and as much as possible. The purpose of UI when using this API should be to gather user consent to the linking of information between sites and then do no more. Account selection, account data storage, policy presentation, and capability selection are all things we do not want to do as a browser as they are difficult and there is already an industry dedicated to solving these challenges. As such, each credential represents a connection to an identity provider, not an identity.
```

Comment:
Overall this sentence feels a bit less applicable to "accounts push" vs. a full new API alternative - do we really think that account selection and account data storage are not things the browser should ever do?

Reply:
I should probably rewrite this paragraph. It's a holdover from the original explainer from last year. Some of the principles are still applicable but the details are wrong.